### DISCUSSION TITLE
More customization in graphs and dashboard panels

### DISCUSSION THREAD

**Discussion by emaugars12** [langfuse/langfuse#3358] [Ideas]
### More customization in graphs and dashboard panels

On the dashboard, one should be able to display whatever panel he wants. And for metrics charts, more choice could be good (time series graph for example).

**Comment by marcklingen** [comment]
Thanks for sharing. Self-serve customizable dashboards would be super useful here.

**Reply by alez007** [reply]
Is this something that will be supported ? Currently I believe the only extra value langsmith has over langfuse is the custom dashboard.

**Reply by marcklingen** [reply]
High up on the backlog post-v3 release, yes

**Comment by marcklingen** [comment]
Main issue for this: https://github.com/orgs/langfuse/discussions/1011
